program/rust/src/time_machine_types.rs

```diff
@@ -75,7 +75,7 @@ pub mod tests {
     fn c_time_machine_size_is_correct() {
         assert_eq!(
         size_of::<TimeMachineWrapper>(),
-        TIME_MACHINE_STRUCT_SIZE.try_into().unwrap(),
+        TIME_MACHINE_STRUCT_SIZE as usize,
```

Comment:
there are two ambiguous type parameters for the `try_into` now. Something that got pulled in with tokio i think 🤷

Reply:
Looks like the right cast anyhow.